src/main/java/edu/unc/lib/boxc/migration/cdm/services/ArchiveProjectService.java

```diff
@@ -0,0 +1,32 @@
+package edu.unc.lib.boxc.migration.cdm.services;
+
+import edu.unc.lib.boxc.migration.cdm.exceptions.InvalidProjectStateException;
+import org.apache.commons.io.FileUtils;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+/**
+ * Service for archiving chompb project(s)
+ * @author krwong
+ */
+public class ArchiveProjectService {
+
+    /**
+     * Archive a list of projects
+     */
+    public void archiveProject(Path currentDirectory, List<Path> projectPaths) throws IOException {
```

Comment:
A few suggestions related to this method:
1. At that moment i think you could use this method to move any directory in the filesystem into the archived folder. It might make sense to change projectPaths to projectName, and just pass in the name of the directory for the project within the currentDirectory.
2. A user could create a project with the word "archived" in the path (we do work with library archives), which could incorrectly be labeled as archived by the status command. It would probably be safer to name the directory `.archived` so it's treated as a hidden directory and less likely to have a name collision. I'd also suggest requiring the path have `/.archive/` in it to be counted.
3. We should probably make a constant out of the name of the archive directory, and use that constant in any other places that need to reference the directory
4. The method should be `archiveProjects` since it archives multiple projects